io.github.chen08209.FlClash.yml

```diff
@@ -0,0 +1,44 @@
+id: io.github.chen08209.FlClash
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: AppRun
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=session-bus
+  - --device=dri
+
+modules:
+  - name: FlClash
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 io.github.chen08209.FlClash.desktop ${FLATPAK_DEST}/share/applications/io.github.chen08209.FlClash.desktop
+      - install -Dm644 io.github.chen08209.FlClash.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/io.github.chen08209.FlClash.png
+      - install -Dm644 io.github.chen08209.FlClash.metainfo.xml ${FLATPAK_DEST}/share/metainfo/io.github.chen08209.FlClash.metainfo.xml
+
+      - chmod +x ./FlClash.AppImage
```

Comment:
why not build from source?

Reply:
It's not clear to me how to build a Flatpak app from source. There are currently no more specific build documents available for this project. So I think it would be simpler to build from an existing binary.